src/service.ts

```diff
@@ -1,16 +1,17 @@
 import express, { Response } from "express";
 import httpContext from "express-http-context";
 import logger from "./logger";
-import { PublicInspectionError, PublicDataValidationError, IEthereumResponse } from "./dataEntities";
+import { PublicInspectionError, PublicDataValidationError, ApplicationError } from "./dataEntities";
 import { Raiden, Kitsune } from "./integrations";
-import { Watcher } from "./watcher";
+import { Watcher } from "./watcher/watcher";
```

Comment:
Perhaps I would add a ./watcher/index.ts to simplify imports.

Reply:
will do